src/ui/organisms/filePicker/i18n/filePicker_en.json

```
@@ -0,0 +1,15 @@
+{
+  "filePicker": {
+    "fileSizeUnit": {
+      "B": "Bytes",
+      "KB": "KB",
```

The metric prefix for `kilo` is [`k`](https://en.wikipedia.org/wiki/Metric_prefix) - (see [kilobyte](https://en.wikipedia.org/wiki/Kilobyte)).

```suggestion
      "KB": "kB",
```